lib/core/cursor.js

```diff
@@ -601,12 +609,124 @@ Cursor.prototype._initializeCursor = function(callback) {
     return;
   }
 
+  function done(err, result) {
```

Comment:
It is very hard to see what changed here b/c of the huge diff. Is there something specific to point to?

Reply:
Nothing changed in this section, it was simply moved outside of the server selection callback so that the handler could be shared with `executeOperation`.